demo/date-picker-basic-demos.html

```diff
@@ -47,9 +47,10 @@ <h3>Date range</h3>
     <p>Use two date pickers to create a date range selector.</p>
     <vaadin-demo-snippet id="date-picker-basic-demos-date-range" when-defined="vaadin-date-picker">
       <template preserve-content>
-        <vaadin-date-picker id="start" label="Start"></vaadin-date-picker>
-        <vaadin-date-picker id="end" label="End"></vaadin-date-picker>
-
+        <vaadin-custom-field label="Date range">
```

Comment:
Let's update the demo to output the custom field value below, same as in the [combo-box demo](https://vaadin.com/components/vaadin-combo-box/html-examples/combo-box-basic-demos#allow-custom-values).

Reply:
Done